### PR TITLE
Modification of SDK camera attributes and update of T_world_cam in Perception

### PR DESCRIPTION
**Pull request by ClaireHzl**
PollenSDKCameraWrapper :
- add the getter of cam name
- add the getter of head orientation (to be used in Perception to update T_world_cam)
- distinction between teleop and depth cam

Perception :
- automatic attribution of T_world_cam if PollenSDKCameraWrapper and dynamic update if cam = 'teleop'

### CODE REVIEW

File: pollen_vision/pollen_vision/camera_wrappers/pollen_sdk_camera/pollen_sdk_camera_wrapper.py [hunk 1]
```diff
@@ -38,24 +39,32 @@ def get_data(self) -> Tuple[Dict[str, npt.NDArray[np.uint8]], Dict[str, float],
...
-                data["right"] = self.right  # type: ignore
-                # fixme
-                data["depthNode_left"] = self.left  # type: ignore
-                data["depthNode_right"] = self.right  # type: ignore
-
-                return data, latency, ts
-            else:
-                self._logger.error("capture failed")
-                return data, latency, ts
+            # cam = getattr(self._reachy.cameras, self._cam_name)
```
Comment (apirrone) [inline pollen_vision/pollen_vision/camera_wrappers/pollen_sdk_camera/pollen_sdk_camera_wrapper.py:42]: You can remove the commented code here

File: pollen_vision/pollen_vision/perception/perception.py [hunk 2]
```diff
@@ -120,33 +160,33 @@ def set_tracked_objects(self, objects: list[str]) -> None:
                 self.tracked_objects.append(obj)
 
 
-if __name__ == "__main__":
-    import FramesViewer.utils as fv_utils
-    from pollen_vision.camera_wrappers.depthai import SDKWrapper
-    from pollen_vision.camera_wrappers.depthai.utils import get_config_file_path
+# if __name__ == "__main__":
```
Comment (apirrone) [inline pollen_vision/pollen_vision/perception/perception.py:163]: Why comment this ? Is this code not working anymore ?

**Review by apirrone — APPROVED** [review]
Looks good! just remove the dead code and uncomment the __main__ example if possible

File: pollen_vision/pollen_vision/perception/perception.py (hunk 2 above)
Comment (ClaireHzl) [reply pollen_vision/pollen_vision/perception/perception.py]: my mistake :)

**Pull request merged**